awx/ui_next/src/screens/User/UserOrganizations/index.js

```diff
@@ -1 +0,0 @@
-export { default } from './UserOrganizations';
```

Comment:
Iet's not move `UserOrganizationsListItem` to it's own directory. 

The convention we've adopted for the rest of the project is that the top-level folders inside of a screen correspond to different routes within that screen.

The new functionality looks good to me, but please change the location of these files back to their original places:

```
├── shared
│   ├── index.js
│   ├── UserForm.jsx
│   └── UserForm.test.jsx
├── UserAdd
│   ├── index.js
│   ├── UserAdd.jsx
│   └── UserAdd.test.jsx
├── UserDetail
│   ├── index.js
│   ├── UserDetail.jsx
│   └── UserDetail.test.jsx
├── UserEdit
│   ├── index.js
│   ├── UserEdit.jsx
│   └── UserEdit.test.jsx
├── User.jsx
├── UserList
│   ├── index.js
│   ├── UserListItem.jsx
│   ├── UserListItem.test.jsx
│   ├── UserList.jsx
│   └── UserList.test.jsx
├── UserOrganizationList
│   ├── index.js
│   ├── UserOrganizationList.jsx
|    └── UserOrganizationListItem.jsx
├── Users.jsx
├── Users.test.jsx
├── UserTeams
│   ├── index.js
│   └── UserTeams.jsx
├── User.test.jsx
└── UserTokens
    ├── index.js
    └── UserTokens.jsx
```